test/parallel/test-whatwg-url-properties.js

```diff
@@ -1,9 +1,11 @@
+// Flags: --expose-internals
 'use strict';
 
 require('../common');
 
 const URL = require('url').URL;
 const assert = require('assert');
+const urlToOptions = require('internal/url').urlToOptions;
```

Comment:
I'm not convinced that it matters too much either way, but there's no harm in separating it out.